modules/fileout_netcdf/FONcTransform.h

```diff
@@ -65,6 +66,7 @@ class FONcTransform: public BESObj {
 	string _returnAs;
 	vector<FONcBaseType *> _fonc_vars;
 	vector<FONcBaseType *> _total_fonc_vars_in_grp;
+    set<string> _included_grp_names;
```

Comment:
The STL 'set' class is sorted by name. If you want to preserve the dataset's ordering of the groups, there's an unordered_set you can use although it might break the CentOS 6 build. I'm going to remove that this Sprint or next sprint, so using unordered_set<> is an option.